build.gradle

```diff
@@ -202,8 +202,3 @@ buildscript {
         classpath 'org.eclipse.jgit:org.eclipse.jgit:2.0.0.201206130900-r'
     }
 }
-
-task wrapper(type: Wrapper) {
```

Comment:
See [gradle's recommendation](https://docs.gradle.org/current/userguide/gradle_wrapper.html)

>     The Wrapper can be further customized by adding and configuring a Wrapper task in your build script, and then executing it.


1. No it doesn't :)
2. Upgrading gradle requires a commit anyway
3. Keeping these in sync is trivial. When performing a gradle upgrade, you simply change the version in build.prop and run the task.